python/ray/train/torch/train_loop_utils.py

```diff
@@ -63,11 +69,64 @@ def get_device() -> Union[torch.device, List[torch.device]]:
         >>> # ray.get_gpu_ids() == [4,5]
         >>> # torch.cuda.is_available() == True
         >>> # get_device() == torch.device("cuda:4")
+
+
+        >>> # You can move model to device by:
+        >>> # model.to(ray.train.torch.get_device())
+        >>> #
+        >>> # instead of manually checking the device type:
+        >>> # model.to("cuda" if torch.cuda.is_available() else "cpu")
     """
     from ray.air._internal import torch_utils
 
     record_extra_usage_tag(TagKey.TRAIN_TORCH_GET_DEVICE, "1")
-    return torch_utils.get_device()
+    return torch_utils.get_devices()[0]
+
+
+@PublicAPI(stability="alpha")
```

Comment:
Let's just start this off as `beta` since it's not a totally new/exploratory API.
```suggestion
@PublicAPI(stability="beta")
```